### PR TITLE
complete card 2, GET /api/topics except for error 404

### PR DESCRIPTION
**Pull request by SimonRosellys**
I have set up two tests that confirm the endpoint works. Tomorrow I will implement some error handling to deal with 404 - path not found etc

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+beforeEach(() => seed(testData));
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toBeInstanceOf(Array);
+        console.log(res.body);
```
Comment (cavalierswain) [inline __tests__/app.test.js:17]: Make sure to remove console logs when you're finished with them

File: controllers/controllers.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+const { getTopicsMod } = require("../models/models");
+
+exports.getTopicsCon = (req, res, next) => {
+  getTopicsMod().then((topics) => {
+    // console.log(topics, "<----- topics in controller");
```
Comment (cavalierswain) [inline controllers/controllers.js]: Also remember to remove comments when finished too!

**Comment by cavalierswain** [comment]
Good work so far Simon! 

Have a look at how you've named your controllers and models, think about readability. 
getTopicsCon & getTopicsMod are quite similar and could get confused. 
getTopics is great for the controller and fetchTopics for the model would be a bit clearer

Lovely work! keep it up

**Comment by SimonRosellys** [comment]
Thank you!

**Pull request merged**